### ISSUE TITLE
Rule proposal: disallow string literal in mustache

### ISSUE THREAD

**Issue by golopot** [vuejs/eslint-plugin-vue#889]
**Please describe what the rule should do:**
Disallow string literals in mustache.

**What category should the rule belong to?**

[x] Enforces code style

**Provide 2-3 code examples that this rule should warn about:**
```vue
<div>{{"foo"}}</div>
<div :class="{{'baz'}}" />
```

**Additional context**
Similar rule in eslint-plugin-react: [react/jsx-curly-presense](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md).
